pyflakes/checker.py

```diff
@@ -663,17 +664,27 @@ def getNodeName(node):
         return node.name
 
 
+class ANY_TYPING_NAME(object):
+    """
+    Marker for asking if the given node is a valid but unspeciffied typing member.
+
+    Note: this is *not* asking if the member is `typing.Any`.
+    """
+
+
 def _is_typing(node, typing_attr, scope_stack):
```

Comment:
the change to this function feels like it has been overloaded to do too much (over-DRYing) -- I would probably split the "ANY" concept to a separate function (it feels to me like the change here is complicating this function to do too much and it's harder to grok what's going on, whereas a separate function would be easier to comprehend)

put in another way, this function would be difficult to type-annotation (it would require a `Union`) which is often an indicator that it is doing too much

Reply:
I've reworked this in 4037b061592b3f69ffbff8ee3c2a79f66d2ec175 so that we pass a name-comparison function in, rather than the target of the comparison. I'm not actually sure if that's better than just duplicating the function, so happy to change it if you think it's better to have the duplication.

Reply:
lg2m